source/API_Reference/Web_API_v3/How_To_Use_The_Web_API_v3/authentication.md

```diff
@@ -31,3 +32,39 @@ Authorization: Bearer Your.API.Key-HERE
 {% codeblock lang:bash %}
 curl -X "GET" "https://api.sendgrid.com/v3/templates" -H "Authorization: Bearer Your.API.Key-HERE" -H "Content-Type: application/json"
 {% endcodeblock %}
+
+{% anchor h3 %}
+On-Behalf of Subuser
+{% endanchor %}
+
+The on-behalf-of header allows you to make calls for a particular subuser through the parent account, this can be useful for automating bulk updates, or administering a subuser without changing authentication in your code. In the header you are passing, you will simply need to add: `On-Behalf-Of: subuser_username`
+ +
+ +This will generate the api call as if it was the subuser account itself making the call. Just make sure you are using the correct subuser username.
+ +
+ +When authenticating using the on-behalf-of header, you will need to use the API key or basic auth credentials of the parent account.
+ +
+ +**Using Basic Auth:**
```

Comment:
We're purposefully *not* documenting how to use basic auth. It's doc best practice to present one option in the doc, and the option we're presenting is API key, so can you please remove this example and the mention above?